src/System.Private.ServiceModel/tools/CertificateGenerator/CertificateGenerator.csproj

```diff
@@ -9,7 +9,6 @@
 
   <ItemGroup>
     <PackageReference Include="BouncyCastle.Crypto.dll" Version="1.8.1" GeneratePathProperty="true" />
-    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" PrivateAssets="All" />
```

Comment:
Yes, I've confirmed that removing this package reference will break compiling the certificate generator tool unless we install the Full Framework SDK version required by the tool on the host machine.

Reply:
It is old project format.

Reply:
Sorry I was missing the "why" above. Why is it?

Reply:
Hmm I think it's in old format because it's created at an early stage when the new project format for full framework project was not so prevalent or supported :) It has been working well so far so we have not prioritized updating it. Maybe we'll consider upgrading the project format sometime later.

Reply:
I think we should have a tracking issue to update it. There's really no reason anymore to use the old project system format.

Reply:
I just remembered we once had a PR to update the project, I'm going to verify the changes and reactivate it for review.